lib/db.js

```diff
@@ -319,25 +332,34 @@ const collectionKeys = [
  * @param {Db~collectionResultCallback} [callback] The collection result callback
  * @return {Collection} return the new Collection instance if not in strict mode
  */
+const collectionSchema = {
+  w: { type: ['number', 'string'] },
+  wtimeout: { type: 'number' },
+  j: { type: 'boolean' },
+  raw: { type: 'boolean' },
+  pkFactory: { type: 'object' },
+  readPreference: { type: [ReadPreference, 'object', 'string'] },
+  serializeFunctions: { type: 'boolean' },
+  strict: { type: 'boolean' },
+  readConcern: { type: 'object' },
+  'readConcern.level': { type: 'object' },
```

Comment:
readConcern.level should be a string

Reply:
Also, I wonder if this just means we should have a custom validator for readConcern

Reply:
agree with this, I think we should also remove the documentation for the sub-property of read concern (and remove the validation of the `level` property).

Reply:
Should I make a ticket to make a custom validator for readConcern?